src/test/resources/vaunt/information-service.json

```diff
@@ -0,0 +1,83 @@
+{
+  "name": "information-service",
+  "capabilities": {
+    "contracts": [
+      {
+        "destinationType": "QUEUE",
+        "destinationName": "request_for_information_queue",
+        "message": {
+          "type": "object",
+          "id": "urn:jsonschema:com:hltech:vaunt:generator:domain:representation:RepresentationWriterSpec:RequestMessage",
```

Comment:
in id Vaunt puts only actual name of the class representing the message, in this case: RequestMessage

Reply:
fixed